server/chat/backend/agent/tools/victorops_tool.py

```
@@ -0,0 +1,106 @@
+    """Check if a user has valid Splunk On-Call credentials stored."""
+    try:
+        from utils.auth.token_management import get_token_data
+
+        creds = get_token_data(user_id, "victorops")
+        return bool(creds and creds.get("api_id") and creds.get("api_key"))
+    except Exception:
+        logger.debug("VictorOps connection check failed for user %s", user_id)
+        return False
+
+
+def _get_client(user_id: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_get_client`

(ANN202)